verilog/analysis/verilog_linter.cc

```diff
@@ -215,10 +234,10 @@ void ViolationFixer::HandleViolation(
   }
 }
 
-ViolationFixer::AnswerChoice ViolationFixer::InteractiveAnswerChooser(
+ViolationFixer::Answer ViolationFixer::InteractiveAnswerChooser(
     const verible::LintViolation& violation, absl::string_view rule_name) {
   static absl::string_view help_message =
-      "y - apply fix\n"
+      "y - apply first fix [or type number of alternative e.g. '2']\n"
```

Comment:
maybe show help on a separate line:

`[2-9]` apply alternative fix

Reply:
Done. Help message will now be generated depending on if there is an alternate fix and only inlcude such line if it is.